src/Lykke.Job.BlockchainOperationsExecutor.Core/ChaosKitty.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Lykke.Job.BlockchainOperationsExecutor.Core
+{
+    public static class ChaosKitty
```

Comment:
Please makr this class not static. Use inject interface of this ChaosKitty to Handlers. Becouse it's good for make unit test to Handler
And please move ChaosKitty to some base nuget packet (maybe common) beciuse it's copy-past
and add extantion to ContainerBuilder:
.InviteChaosKitty(settingsChaosKitty)  for register this interface and class